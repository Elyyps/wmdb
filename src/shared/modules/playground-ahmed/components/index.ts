export * from "./playground-ahmed.component";
