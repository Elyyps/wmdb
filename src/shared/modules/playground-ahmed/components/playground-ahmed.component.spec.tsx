import React from "react";
import { shallow } from "enzyme";
import { PlaygroundAhmedComponent } from "./playground-ahmed.component";

describe("[PlaygroundAhmedComponent]", () => {
  it("should render without crashing", () => {
    const renderedComponent = shallow(<PlaygroundAhmedComponent />);

    expect(renderedComponent).toMatchSnapshot();
  });
});
