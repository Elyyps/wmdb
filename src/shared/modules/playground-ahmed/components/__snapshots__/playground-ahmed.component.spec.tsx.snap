// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[PlaygroundAhmedComponent] should render without crashing 1`] = `
<div>
  <div
    className="main"
  >
     Playground Ahmed 
  </div>
  <div>
     Hello World 
  </div>
</div>
`;
